RubeesEat/Pages/ShowBills.cshtml

```diff
@@ -29,11 +29,11 @@
                     <span class="amount @(item.Amount < 0 ? "negative-amount" : "positive-amount")" t-id="amount">
                         @item.Amount€
                     </span>
-                    <span class="description">
+                    <span class="description" t-id="description">
                         @item.Description
                     </span>
                 </div>
-                <div class="date">
+                <div class="date" t-id="date">
                     @item.Date.ToString("D", CultureInfo.CreateSpecificCulture("en-US"))
                 </div>
```

Comment:
Same here. `description` and `date` should not have any styles because we define the styles in `BillDetails.css`. If that is true we can remove them here.

Reply:
Okay then I leave it up to you. we can either break the styling now or remember to fix this when we switch to bootstrap.